cdc/puller/frontier/frontier.go

```diff
@@ -63,33 +76,50 @@ func (s *spanFrontier) Frontier() uint64 {
 }
 
 // Forward advances the timestamp for a span.
-func (s *spanFrontier) Forward(span regionspan.ComparableSpan, ts uint64) {
-	s.insert(span, ts)
+func (s *spanFrontier) Forward(regionID uint64, span regionspan.ComparableSpan, ts uint64) {
+	// it's the fast part to detect if the region is split or merged,
+	// if not we can update the minTsHeap with use new ts directly
+	if n, ok := s.cachedRegions[regionID]; ok && n.regionID > 0 && n.end != nil {
```

Comment:
What is the meaning is regionID == 0? Could you please use a `commented const` instead of use `0` for the special case?

Reply:
done